src/main/java/org/maxgamer/rs/command/commands/Warp.java

```diff
@@ -78,6 +62,31 @@ public void execute(Player p, String[] args) throws Exception {
         p.sendMessage("Warped to " + d.name);
     }
 
+    /**
+     * Add a warp based on a "::warp add <my warp name>" command
+     * @param p the player
+     * @param args the message they wrote
+     */
+    protected void add(Player p, String[] args) {
+        if (args.length < 2) {
+            p.sendMessage("Usage ::warp add [name of warp]");
+            return;
+        }
+
+        Destination dest = new Destination();
+        Location l = p.getLocation();
+        dest.x = l.x;
+        dest.y = l.y;
+        dest.z = l.z;
+
+        dest.name = args[1].toLowerCase();
+        for (int i = 2; i < args.length; i++) dest.name += " " + args[i];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Use a StringBuilder instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1643)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS00122)